### ISSUE TITLE
TrainingArgs not recognised in Trainer

### ISSUE THREAD

**Issue by PrithivirajDamodaran** [huggingface/setfit#559]
v1.1.0

Below warnings are throws by the snippet. 

  >2024-09-22 19:09:50,723 - No `TrainingArguments` passed, using `output_dir=tmp_trainer`.
  2024-09-22 19:09:50,735 - No `loss` passed, using `losses.CoSENTLoss` as a default option.

```python

from setfit import SetFitModel, Trainer, TrainingArguments

:
:
training_args = TrainingArguments(
            output_dir=output_dir,                
            eval_strategy=save_strategy, 
            save_strategy=save_strategy,          
            batch_size=batch_size, 
            num_epochs=epochs,
            body_learning_rate = lr,         
            warmup_proportion=warmup_proportion,   
            logging_dir=f"{output_dir}/logs",    
            load_best_model_at_end=True,  
            show_progress_bar = True,
            use_amp = use_amp,
            samples_per_label=min_samples,
            loss=CosineSimilarityLoss,
        )

 trainer = Trainer(
        model=model,
        args=training_args,
        train_dataset=train_dataset,
        eval_dataset=test_dataset,
        metric="accuracy",
        column_mapping={"text": "text", "label": "label"},
    )
```

**Comment by seanfarr788** [comment]
I think it is just an erroneous warning message, printing out trainer.args shows the args being set correctly
